### ISSUE TITLE
Plugin tries to look for dir in node_modules/.bin/ when relative path is specified

### ISSUE THREAD

**Issue by Bewinxed** [kravetsone/elysia-autoload#2]
I am using Elysia in a sveltekit project, with the `svelte-adapter-bun` adapter.

Here's the Repo
https://github.com/Bewinxed/elysiakit

I am initiating it like so:
```typescript
export const app = new Elysia({})
	.use(
		autoload({
			prefix,
			dir: "./routes",
			types: {
				output: "/src/lib/elysia/route-types.ts",
			},
		}),
	)
	.use(
		swagger({
			path: `${prefix}/docs`,
		}),
	)
	.onRequest(({ request, set }) => {
		// FIX: For elysia-autoload, using swagger and elysia-autoload together causes double prefix, so we remove it
		if (request.url.includes(`${prefix}${prefix}`)) {
			set.redirect = request.url.replace(`${prefix}${prefix}`, prefix);
			return request;
		}
	});
```

here's my tsconfig
```
{
	"extends": "./.svelte-kit/tsconfig.json",
	"compilerOptions": {
		"lib": [
			"ESNext"
		  ],
		  "module": "ESNext",
		  "target": "ESNext",
		"allowJs": true,
		"checkJs": true,
		"esModuleInterop": true,
		"forceConsistentCasingInFileNames": true,
		"resolveJsonModule": true,
		"skipLibCheck": true,
		"sourceMap": true,
		"strict": true,
		"moduleResolution": "Bundler",
		"allowSyntheticDefaultImports": true,
		"noEmit": true,
		"paths": { "/*": ["./*"] }
	},
	"include": [
		"src"
	]
	// Path aliases are handled by https://kit.svelte.dev/docs/configuration#alias
	//
	// If you want to overwrite includes/excludes, make sure to copy over the relevant includes/excludes
	// from the referenced tsconfig.json - TypeScript does not merge them in
}
```
I run my project like  bunx --bun vite

but when the plugin is loaded, the process.argv[1] resolves to `/root/dev/elysiakit/node_modules/.bin/routes`

```
/root/dev/elysiakit
22 | const TYPES_OUTPUT_DEFAULT = "./routes-types.ts";
23 | const TYPES_TYPENAME_DEFAULT = "Routes";
24 | async function autoload({ pattern, dir, prefix, schema, types, } = {}) {
25 |     const directoryPath = (0, utils_1.getPath)(dir || "./routes");
26 |     if (!(0, node_fs_1.existsSync)(directoryPath))
27 |         throw new Error(`Directory ${directoryPath} doesn't exists`);
                   ^
error: Directory /root/dev/elysiakit/node_modules/.bin/routes doesn't exists
      at /root/dev/elysiakit/node_modules/elysia-autoload/dist/index.js:27:15
      at autoload (/root/dev/elysiakit/node_modules/elysia-autoload/dist/index.js:24:27)
error: script "dev" exited with code 1
```

Any idea why this is happening?

**Comment by kravetsone** [comment]
maybe it cause because https://github.com/kravetsone/elysia-autoload/blob/d8b4ad25fe9869a1da90e74eed1e3d61a13dd7c1/src/utils.ts#L5 ...

bunx --bun vite is executed by node_modules/.bin/vite

any idea how to solve it?

send me yours argv
```ts
console.log(process.argv);
```

**Comment by Bewinxed** [comment]
> maybe it cause because
> 
> https://github.com/kravetsone/elysia-autoload/blob/d8b4ad25fe9869a1da90e74eed1e3d61a13dd7c1/src/utils.ts#L5
> 
> ...
> bunx --bun vite is executed by node_modules/.bin/vite
> 
> any idea how to solve it?
> 
> send me yours argv
> 
> ```ts
> console.log(process.argv);
> ```

The output is
```
[ "/root/.bun/bin/bun", "/root/dev/elysiakit/node_modules/.bin/vite" ]
```

**Comment by kravetsone** [comment]
> The output is
> 
> ```
> [ "/root/.bun/bin/bun", "/root/dev/elysiakit/node_modules/.bin/vite" ]
> ```

and send me import.meta.path

**Comment by kravetsone** [comment]
> ```ts
> // FIX: For elysia-autoload, using swagger and elysia-autoload together causes double prefix, so we remove it
> 		if (request.url.includes(`${prefix}${prefix}`)) {
> 			set.redirect = request.url.replace(`${prefix}${prefix}`, prefix);
> 			return request;
> 		}
> ```
		
can you tell more about it? double slash in / route is fixed now with https://github.com/kravetsone/elysia-autoload/commit/405dad693de1d4493b3209105405e9e319d37435

**Comment by Bewinxed** [comment]
> > ```ts
> > // FIX: For elysia-autoload, using swagger and elysia-autoload together causes double prefix, so we remove it
> > 		if (request.url.includes(`${prefix}${prefix}`)) {
> > 			set.redirect = request.url.replace(`${prefix}${prefix}`, prefix);
> > 			return request;
> > 		}
> > ```
> 
> can you tell more about it? double slash in / route is fixed now with [405dad6](https://github.com/kravetsone/elysia-autoload/commit/405dad693de1d4493b3209105405e9e319d37435)

I opened another issue for this [here](https://github.com/kravetsone/elysia-autoload/issues/3)


> > The output is
> > ```
> > [ "/root/.bun/bin/bun", "/root/dev/elysiakit/node_modules/.bin/vite" ]
> > ```
> 
> and send me import.meta.path

import meta inside utils.js **/root/dev/elysiakit/node_modules/elysia-autoload/dist/utils.js**
import meta inside src/elysia/backend.ts **undefined**

**Comment by kravetsone** [comment]
> import meta inside src/elysia/backend.ts **undefined**

hmm... thats strange

but you can use absolute import for now
